make/harbor.yml

```diff
@@ -161,3 +161,6 @@ proxy:
     - core
     - jobservice
     - clair
+
+# Comma-separated adapters for replication. All adapters are supported if not specify 
+# replication_adapters:
```

Comment:
Provide a full list sample in the comment
#replication_adapters: docker-registry, docker-hub- docker-registry,docker-hub,harbor,helm-hub,google-gcr,aws-ecr

Reply:
+1 I'd suggest we provide a full list of yaml and comment the ones we don't support by default, so user knows how to enable them on their own risk.